style.css

```diff
@@ -0,0 +1,77 @@
+* {
+  font-family: 'Poppins', sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+li {
+  list-style-type: none;
+}
+
+body {
+  display: flex;
+  max-width: 375px;
```

Comment:
- [ ] As you have max-width defined on your body as 375px, your design is not responsive above 375px.
Kindly consider making your design responsive. it is part of the requirement and also mentioned by the previous reviewer.
- Minimum mobile screen size should be `375`px. Errors in screen sizes less than `375`px should be ignored. Otherwise, the mobile version should be responsive until the desktop screen size.